src/Listener/SaveSuspensionToDatabase.php

```diff
@@ -20,6 +21,21 @@ class SaveSuspensionToDatabase
     use AssertPermissionTrait;
 
     /**
+     * @var SuspendValidator
+     */
+    protected $validator;
+
+    /**
+     * SaveSuspensionToDatabase constructor.
```

Comment:
That will either remind us to write something meaningful later, or we can safely ignore it here.